spec/spec_helper.rb

```diff
@@ -2,6 +2,7 @@
 require 'simplecov'; SimpleCov.start { add_filter "_spec" }
 require 'boxr'
 require 'awesome_print'
+require 'boxr_spec'
 
 RSpec.configure do |config|
   config.before(:each) do
```

Comment:
Here to avoid resetting data in Box for the validator specs, I added this snippet locally:

```suggestion
  config.before(:each) do
    if test.metadata[:skip_reset]
      puts "Skipping reset"
      next
    end

```